db/model/token.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2016, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/token.js
+ */
+
+const constants = require('../constants');
+const common = require('../helpers/common');
+const u = require('../helpers/userUtils');
+
+const assoc = {};
+
+module.exports = function token(seq, dataTypes) {
+  const Token = seq.define('Token', {
+    id: {
+      type: dataTypes.UUID,
+      primaryKey: true,
+      defaultValue: dataTypes.UUIDV4,
+    },
+    name: {
+      type: dataTypes.STRING(constants.fieldlen.normalName),
+      allowNull: false,
+    },
+    token: {
+      type: dataTypes.STRING,
+      allowNull: false,
+      unique: true,
+    },
```

Comment:
@iamigo @pallavi2209 Any thoughts on having a default values for all the data types? We might be able to get ride of the responsify function at one point of time.